scripts/pysa_fuzzer/build_script.py

```diff
@@ -0,0 +1,136 @@
+import subprocess
+import os
+import json
+import argparse
+from fuzzer2 import CodeGenerator
+
+def run_command(command):
+    process = subprocess.Popen(command, shell=True)

```

Comment:
We try to avoid `shell=True` since it can lead to security vulnerabilities. Obviously this script is not likely to lead to vulnerabilities, but for the sake of consistency, I would still avoid `shell=True` here.
Please also use `subprocess.run`, we should only use `subprocess.Popen` for the most complex use case.
And also, use `check=True`, otherwise you are ignoring errors.